cegs_portal/search/templates/search/v1/partials/_experiment_index.html

```diff
@@ -0,0 +1,91 @@
+                </div>
+                <div id="categorical-facets">
+                    {% for facet, values in facets.items %}
+                    <fieldset class="my-4" name="facetfield">
+                        <legend class="font-bold">{{ facet }}</legend>
+                        <div class="flex flex-row flex-wrap gap-1">
+                            {% for value in values %}
+                                <div class="ml-1">
+                                    <input id="{{ value.id }}" type="checkbox" name="{{ facet }}"/>
+                                    <label for="{{ value.id }}">{{ value.value }}</label>
+                                </div>
+                            {% endfor %}
+                        </div>
+                    </fieldset>
+                    {% endfor %}
+                </div>
+            </div>
+        </div>
+
+        <div id="experiment-list" class="basis-3/4">
+            {% for experiment in experiments %}
+            <a href="{% url 'search:experiment' experiment.accession_id %}" class="exp-list-content-container-link experiment-summary" data-accession="{{ experiment.accession_id }}" data-name="{{ experiment.name }}">
+                <div class="container items-center">
+                    <div class="flex justify-between">
+                        <div class="exp-name">{{ experiment.name }}</div>
+                    </div>
+                    <div>{{ experiment.description }}</div>
+                    <div class="flex justify-between">
+                        <div class="cell-lines">Cell Lines: {{ experiment.cell_lines }}</div>
+                        <div class="accession-id">{{ experiment.accession_id }}</div>
+                    </div>
+                </div>
+            </a>
+            {% endfor %}
+        </div>
+        </div>
+    </div>
+</div>
+</div>
+
+{% endblock %}
+
+{% block inline_javascript %}
+<script type="module">
+    import { facetFilterSetup } from "{% static 'search/js/experiment_list/facet_filter.js' %}";
+</script>
+<script>
+    function closeModal() {
+        let modal = document.getElementById("exp-modal");
+        modal.classList.add("closing") // Kicks off an animation on the modal dialog
+        modal.addEventListener("animationend", (event) => {
+            modal.remove(); // remove the dialog from the DOM once the animation is over
+        })
+    }
+</script>
+<script type="module">
+    import { facetFilterSetup } from "{% static 'search/js/experiment_list/facet_filter.js' %}";
+    import { addDragListeners, addDropListeners, addSelectListeners } from "{% static 'search/js/experiment_list/drag_drop.js' %}";
+    facetFilterSetup();
+    addDropListeners();
+    addDragListeners();
+    addSelectListeners();
+</script>
+
+{% if logged_in %}
+<script type="module">
+    import { downloadDataSetup } from "{% static 'search/js/experiment_list/download_data.js' %}";
+    downloadDataSetup("{{ csrf_token }}");
+</script>
+{% endif %}
```

Comment:
I don't think users should be able to download stuff from the modal. I think this means you don't need to pass "logged_in" to the template from the view.